apps/govquests-api/rails_app/db/migrate/20241001131854_create_quests.rb

```diff
@@ -0,0 +1,13 @@
+class CreateQuests < ActiveRecord::Migration[8.0]
+  def change
+    create_table :quests do |t|
+      t.string :img_url
+      t.string :title
+      t.string :reward_type
+      t.text :intro
+      t.text :steps
```

Comment:
this project is using postgres or sqlite? If you're using postgres, it's better to use a jsonb column for better performance and querying capabilities